### DISCUSSION TITLE
Using net8 preview 5 with Visual Studio for Mac

### DISCUSSION THREAD

**Discussion by KeithBoynton** [dotnet/maui#16017] [Q&A] [answered]
I've been trying to use net8 preview 5 with Visual Studio for Mac but can't seem to get it to work. The problem seems to be that I can't find where to download Visual Studio for Mac Preview version 17.7 all the links seem to just give me version 17.6.

Has anyone successfully done this yet?

**Comment by DDHSchmidt** [comment]
@davidortinau basically answered this question [on Twitter already](https://twitter.com/davidortinau/status/1674944723788738567). 

> VSMac doesn’t yet support net8. Use VS in Parallels Win or vs code.

**Comment by mhrastegari** [comment]
I've done it with VS Code using [.NET Meteor](https://[marketplace.visualstudio.com/items?itemName=nromanov.dotnet-meteor](https://marketplace.visualstudio.com/items?itemName=nromanov.dotnet-meteor)) and C# dev kit

https://twitter.com/mhrastegari77/status/1666924286085324803?s=20

**Comment by KeithBoynton** [comment, marked as answer]
If anyone comes across this, you can now use NET8 preview with the latest stable VS for Mac by using the Preview Features in settings. It's working fine for me

**Comment by cgp1976** [comment]
I am getting .NET 9.0 SDK is not compatible with this version of Visual Studio for Mac.  my mac vs studio version 17.6.12 (build 410).

kindly advise.
